driver/Dockerfile.hail

```diff
@@ -40,13 +40,14 @@ RUN apt-get update && \
     pip --no-cache-dir install \
         analysis-runner \
         bokeh \
-        cloudpathlib[all] \
+        'cloudpathlib[all]' \
         cpg-utils \
         cpg-workflows \
         gcsfs \
         hail \
         pyarrow \
         sample-metadata \
         metamist \
-        selenium>=3.8.0 \
-        statsmodels
+        'selenium>=3.8.0' \
+        statsmodels && \
+    rm -rf /root/.cache /tmp/tmp*
```

Comment:
It would be good to add a comment here describing what these cache files are and why they should be deleted.